content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
```

Comment:
Need to remove this command from the beginning. We moved this to the end after doing all the other cleanup commands.